web-registry/src/pages/Certificate/Certificate.utils.ts

```diff
@@ -0,0 +1,58 @@
+import { CertificateType } from 'web-components/lib/components/certificate/certificate.types';
+import { truncate } from 'web-components/lib/utils/truncate';
+
+import { getHashUrl } from 'lib/block-explorer';
+import { NormalizedRetirement } from 'lib/normalizers/retirements/normalizeRetirement';
+
+import { CERTIFICATE_TITLE } from './Certificate.constants';
+
+/* getCertificateData */
+
+export type GetCertificateDataParams = {
+  retirement: NormalizedRetirement;
+};
+
+export const getCertificateData = ({
+  retirement,
+}: GetCertificateDataParams): CertificateType => {
+  return {
+    certificateTitle: CERTIFICATE_TITLE,
+    creditsUnits: Number(retirement.amountRetired),
+    creditUnitName: retirement.creditClassName,
+    equivalentTonsCO2: Number(retirement.amountRetired),
```

Comment:
shouldn't we display this only if these are carbon credits?

Reply:
Good catch, I actually prepare the data to do that but did not finish that part, will update it.

Reply:
Fix in: [196e4ac](https://github.com/regen-network/regen-web/pull/1953/commits/196e4ac5d342ccfe5666623239c6ad187f92da80)